dom.go

```diff
@@ -57,13 +57,20 @@ func Element(name string, attrs []Attribute, children ...Node) Node {
 	}
 }
 
-// Text is a helper function to construct a Node that has no tag no attributes and no children.
-func Text(s string) Node {
+// InnerText is a helper function to construct a Node that has no tag no attributes and no children.
+func InnerText(s string) Node {
 	return Node{
 		InnerText: s,
 	}
 }
 
+// InnerHTML is a helper function to construct a Node that has no tag no attributes and no children.
+func InnerHTML(s string) Node {
+	return Node{
+		InnerHTML: template.HTML(s),
+	}
+}
```

Comment:
There's already html function for `<html>`, so we name this `InnerHTML` to match the attribute we are setting, i.e. `InnerHTML` which then raises the question about `Text` setting `InnerText`.

To be unambiguous, I think InnerHTML and innerText are the right names for these functions